### ISSUE TITLE
New .bib web addresses need tweaking

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#291]
They don't point to the right website. I can do this either now or will finish up on Wednesday.

**Comment by aaronmberger** [comment]
When the pointed to the right web address they ran off the page because of length.  I didn't want to spend time figuring out how to make a shortened hyperlink so I just put them in 'manually'.  If you can figure out how to do a shortened name for the hyperlink then sounds good.

**Comment by aaronmberger** [comment]
Michelle had lots of comments about our reference section being incomplete.  This was my quick attempt to get them at least useful.

**Comment by andrew-edwards** [comment]
Fixed in commit 18e0987 . Basically put { } round lots of http;, removed a few page numbers (we haven't been consistent so can maybe fix that next year), and had to add \protect a couple of times. 

Oh - I meant when you hover they often included part of the next line (or the page numbers). Have fixed that. Agree, no need to link to each one individually, just the main website.

Well, the references were the same as for previous years! I realised they were incomplete last year but had no time - we can maybe get them better for next year. Think it's a good sign that people are picking up on somewhat minor things that they hadn't worried about or noticed before!
